js/app/src/components/Audio/Audio.test.ts

```diff
@@ -18,10 +20,14 @@ const loading_status = {
 };
 
 describe("Audio", () => {
+	beforeAll(() => {
+		window.HTMLMediaElement.prototype.play = vi.fn();
+		window.HTMLMediaElement.prototype.pause = vi.fn();
+	});
```

Comment:
I tired to mock this in a slightly more clever way but it didn't work at all, this does work but I'd like to figure out a batter way to do it in the future.